db/src/db.rs

```diff
@@ -29,9 +29,12 @@ impl RocksDB {
         let cf_names: Vec<_> = (0..columns).map(|c| c.to_string()).collect();
 
         let (mut opts, cf_descriptors) = if let Some(ref file) = config.options_file {
-            let mut full_opts = FullOptions::load_from_file(file, None, false).map_err(|err| {
-                internal_error(format!("failed to load the options file: {}", err))
-            })?;
+            let mut full_opts = FullOptions::load_from_file(
+                file,
+                config.cache_size.or_else(|| Some(128 << 20)),
```

Comment:
How could I disable `cache_size`?

If I set `[db] cache_size = null`, the db still set `cache_size = 128 << 20`.

Reply:
We should not allow users to disable the rocksdb cache, user should lower this value (8MB for example), if he wants to reduce memory usage.

Reply:
Why don't allow users to disable the RocksDB cache?

Reply:
How about set it to `None` when `cache_size == 0`?

Reply:
user should modify the `default.db-options` by appending this line if he wants the full control of rocksdb block cache disable / enable:
```
[TableOptions/BlockBasedTable "default"]
no_block_cache=true
```

We chose to put the `cache size` in ckb.toml only because the `default.options` file does not support this parameter

Reply:
But the cache is allocated before the RocksDB load the options file, and couldn't free.

Ref:
- [Why in our `rust-rocksdb`, we couldn't free the cache.](https://github.com/nervosnetwork/rust-rocksdb/pull/18/files#r457146149)
- The [`C++` API `LoadOptionsFromFile(..)`](https://github.com/facebook/rocksdb/blob/0915c99f01b46f50af8e02da8b6528156f584b7c/include/rocksdb/utilities/options_util.h#L71-L79).

Reply:
Understood, I modified the code, and added more comment to the ckb configuration file, please review again, thanks.